test/ModuleInterface/if-configs.swift

```diff
@@ -0,0 +1,58 @@
+// RUN: %target-swift-frontend -emit-interface-path %t.swiftinterface -enable-resilience -emit-module -o /dev/null %s
```

Comment:
Can you do a version that goes to a swiftmodule first too, like test/Serialization/function-default-args.swift?

Reply:
This was tricky because the loaded-from-module interface prints the functions out in alphabetical order.

Reply:
I thought I fixed that, but even if not it's still worth it!